docs/UserGuide.md

```diff
@@ -283,32 +283,8 @@ Shows a list of all employees in the contact list.
 
 
 Format: `c-list`
-
-#### 3.3 Listing all employees who are working today: `c-today`
-
-Shows a list of all active(unarchived) employees whose tags contain today's day (i.e. `Wednesday`, `Tuesday`, etc).
-
-Format: `c-today`
-
-* The search is case-insensitive. e.g `Friday` will match `friday`.
-
-Examples:
```

Comment:
For the example here, maybe can use the format as Example: (Steps + Expected Outcome)? To be consistent with the other parts you have written